src/css/custom.css

```diff
@@ -1,53 +1,56 @@
-/* stylelint-disable docusaurus/copyright-header */
-* {
-  list-style: none;
+body a:hover {
+  color: inherit;
 }
 
-html, body {
-  overflow: hidden;
+.prism-code > * {
```

Comment:
It may look like all of this is being added, but if you look in the diff on the left side, it's all there, it's just been rearranged and git isn't picking up on it